examples/jsm/loaders/OBJLoader2Parallel.d.ts

```diff
@@ -1,23 +1,19 @@
-import {
-	LoadingManager
-} from '../../../src/Three';
-import { OBJLoader2 } from './OBJLoader2';
-
-import { WorkerExecutionSupport } from './obj2/worker/main/WorkerExecutionSupport';
-
 export class OBJLoader2Parallel extends OBJLoader2 {
-
-	constructor( manager?: LoadingManager );
-	preferJsmWorker: boolean;
-	executeParallel: boolean;
-	workerExecutionSupport: WorkerExecutionSupport;
-
-	setJsmWorker( preferJsmWorker: boolean, jsmWorkerUrl: URL ): this;
-	setExecuteParallel( executeParallel: boolean ): this;
-	getWorkerExecutionSupport(): object;
-	buildWorkerCode(): object;
-
-	// @ts-ignore
-	parse( content: ArrayBuffer ): void;
-
+    static OBJLOADER2_PARALLEL_VERSION: string;
+    static DEFAULT_JSM_WORKER_PATH: string;
+    constructor(manager: any);
+    preferJsmWorker: boolean;
+    jsmWorkerUrl: URL;
+    executeParallel: boolean;
+    workerTaskManager: WorkerTaskManager;
+    taskName: string;
+    setExecuteParallel(executeParallel: boolean): OBJLoader2Parallel;
+    setWorkerTaskManager(workerTaskManager: WorkerTaskManager, taskName?: string): OBJLoader2Parallel;
+    setJsmWorker(preferJsmWorker: boolean, jsmWorkerUrl: URL): OBJLoader2Parallel;
+    setTerminateWorkerOnLoad(terminateWorkerOnLoad: boolean): OBJLoader2Parallel;
+    terminateWorkerOnLoad: boolean;
+    private _buildWorkerCode;
+    _executeWorkerParse(content: any): void;
 }
+import { OBJLoader2 } from "./OBJLoader2.js";
```

Comment:
Because I was lazy and now use the typescript compiler to generate the d.ts files from jsdoc (see https://github.com/mrdoob/three.js/pull/19650#issuecomment-654444532 ).

Reply:
I see... And jsdoc can't be configured?

Reply:
The function param types defined in jsdoc are used by `tsc` to specify the types in `d.ts`. I overlooked that the import and export statements now contain the `.js` file extension. This is wrong. I either need to manually correct this or find a way to configure `tsc` to automatically create the correct output (preferred).